third-party/llvm/Makefile

```diff
@@ -12,6 +12,12 @@ else
   CHPL_LLVM_DEBUG := -DCMAKE_BUILD_TYPE=Release
 endif
 
+ifndef CHPL_LLVM_TARGETS_TO_BUILD
+  CHPL_LLVM_TARGETS_TO_BUILD := "host;X86;AArch64;NVPTX;AMDGPU"
+else
+  CHPL_LLVM_TARGETS_TO_BUILD := "$(CHPL_LLVM_TARGETS_TO_BUILD)"
```

Comment:
This just sets the variable to the same value it already had, right?

Reply:
Yes it does, but it also quotes them. Without it, something like `export CHPL_LLVM_TARGETS_TO_BUILD='host;X86'` doesn't work right